app/controllers/tenants_controller.rb

```diff
@@ -1,5 +1,6 @@
 class TenantsController < ApplicationController
   def index
+    @tenants = Tenant.all.where(active: true).where(approved: true)
```

Comment:
Don't we want all the tenants actually? Isn't this only visible to the Platform Admin, who needs to be able to approve and activate/deactivate tenants?